src/main/java/com/keyboardbuilder/enterprise/services/BuildServiceStub.java

```diff
@@ -14,51 +15,34 @@
     private IBuildDAO buildDAO;
 
     @Autowired
-    private IKeyboardBuidDAO keyboardBuildDAO;
-
-
-    public BuildService() {
-
-    }
-
-    public BuildService(IBuildDAO buildDAO) {
-
-        this.buildDAO = buildDAO;
-    }
+    private IKeyboardBuildDAO keyboardBuildDAO;
 
 
     @Override
     @Cacheable(value="build", key="#id")
-    public Build fetchById(int id) {
-        Build foundBuild = buildDAO.fetch(id);
-        return foundBuild;
+    public Build fetchById(int id) throws Exception {
+        return this.buildDAO.fetch(id);
```

Comment:
removed unnecessary variable declaration since the DAO call can be returned up the call chain.